File: chroma-league/src/main/java/net/booone/chromaleague/hud/animations/AllyElderDragonKillAnimation.java
package net.booone.chromaleague.hud.animations;

import net.booone.chromaleague.rest.eventdata.DragonType;

public class AllyElderDragonKillAnimation extends StaticBlinkingAnimation {
    public AllyElderDragonKillAnimation() {
        super(DragonType.ELDER.getColor());
    }
}

File: src/hud/animations/AllyElderDragonKillAnimation.ts
import {DragonType, dragonTypeGetColor} from '../../rest/eventdata/DragonType.js';
import {StaticBlinkingAnimation} from './StaticBlinkingAnimation.js';

export class AllyElderDragonKillAnimation extends StaticBlinkingAnimation {
    constructor() {
        super(dragonTypeGetColor(DragonType.ELDER));
    }
}
